### DISCUSSION TITLE
Search functionality in Wikis

### DISCUSSION THREAD

**Discussion by loreanvictor** [community/community#6888] [Other Feature Feedback, Questions, & Ideas]
I think it would be pretty useful if users could search for specific stuff in Wikis. Technically speaking, Github repo search API already does a decent job for this purpose (I've set it as the default for [CODEDOC](https://codedoc.cc), so you can test it out on any website using it), so it should be fairly straight-forward to implement this.

**Comment by idavidmcdonald** [comment]
Just to add that we would particularly like this and it is something that is making us consider moving away from Github for our team manual. 

Our current workaround is we have put a link on the front page of our wiki pointing to the Github search filtered by wikis and our repository name. 

<img width="588" alt="image" src="https://github.com/community/community/assets/7228605/6fdafa96-a9c7-44eb-b251-125c353edafb">
